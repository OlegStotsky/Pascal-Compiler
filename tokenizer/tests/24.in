3.15e+2