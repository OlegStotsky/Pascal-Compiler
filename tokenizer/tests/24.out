    FLOAT_CONST	        3.15e+2               0               0               0      315,000000
            EOF	            EOF               0               7               0        0,000000
